migrate-aurora-to-tidb.md

```diff
@@ -124,33 +128,33 @@ If you need to enable TLS in the TiDB cluster, refer to [TiDB Lightning Configur
 
 ### Step 4. Import full data to TiDB
 
-1. Create the tables in the target database using TiDB Lightning:
+1. Pass the SecretKey and AccessKey of the account that has access to this Amazon S3 backend storage into the TiDB Lightning node as environment variables. TiDB Lightning also supports reading credential files from `~/.aws/credentials`. When running `tidb-lightning`, if you start the program directly from the command line, it might exit due to the `SIGHUP` signal. It is recommended to use a tool such as `nohup` or `screen`. Refer to the following steps 2 and 3 for details.
```

Comment:
```suggestion
1. Pass the SecretKey and AccessKey of the account that has access to this Amazon S3 storage path into the TiDB Lightning node as environment variables. TiDB Lightning also supports reading credential files from `~/.aws/credentials`. When running `tidb-lightning`, if you start the program directly from the command line, it might exit due to the `SIGHUP` signal. It is recommended to use a tool such as `nohup` or `screen`. Refer to the following steps 2 and 3 for details.
```